pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -668,6 +669,7 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(clusterName string, service *api
 
 		for _, pool := range poolsList {
 			poolIDs = append(poolIDs, pool.ID)
+            monitorIDs = append(monitorIDs, pool.MonitorID)
```

Comment:
excellent. thanks, anguslees. i've run the code through gofmt, amended my commit, and re-pushed.